resources-generator/src/main/kotlin/dev/icerock/gradle/generator/ImagesGenerator.kt

```diff
@@ -93,7 +94,7 @@ abstract class ImagesGenerator(
         private val mrSettings: MRGenerator.MRSettings
     ) : ResourceGeneratorFeature<ImagesGenerator> {
         private val stringsFileTree = info.commonResources.matching {
-            it.include("MR/images/**/*.png", "MR/images/**/*.jpg")
+            it.include("MR/images/**/*.png", "MR/images/**/*.jpg", "MR/images/**/*.svg")
```

Comment:
at now i see support on android and apple, but how it works on web and jvm?

Reply:
I've updated the samples, only the JVM part is not working. Because SVG is not supported in native. Maybe with [Batik](https://xmlgraphics.apache.org/batik/) the JVM part can be finished as well. WDYT?

Reply:
i see that you integrate Batik. i think it will be ok